### PR TITLE
web installer

### PR DESCRIPTION
**Pull request by Zetvue**

### CODE REVIEW

**Comment by Zetvue** [comment]
one second i need to add mods

**Comment by Zetvue** [comment]
ready to merge

**Comment by KTibow** [comment]
hmm maybe, maybe not
mart is coming out soonish, but i guess having people know about cloudclient is good now
however people dont know what chromium is, change dat

**Comment by Zetvue** [comment]
do you really believe people are that stupid to not know what chromium is

**Comment by KTibow** [comment]
> do you really believe people are that stupid to not know what chromium is

YES. Especially in the context of Skyclient, for people who can't figure out how to install Forge.

File: thread.txt [hunk 1]
```diff
@@ -87,7 +87,7 @@ This isn't an issue for Java devs or mod makers because Windows doesn't run the
 
 [SIZE=5][U][URL='https://github.com/SkyblockClient/SkyblockClient/releases/latest']Windows Only[/URL]
 [URL='https://github.com/koxx12-dev/Skyclient-installer-Java/releases/latest']Universal (requires Java)[/URL]
-[URL='https://skyclient.co/cloudclient']Web Installer (requires Chromium)[/URL][/U][/SIZE]
+[URL='https://skyclient.co/cloudclient']Web Installer (requires a modern web browser)[/URL][/U][/SIZE]
```
Comment (KTibow) [inline thread.txt]:
```suggestion
[URL='https://skyclient.co/cloudclient']Web Installer (requires Chrome/Edge, subject to change)[/URL][/U][/SIZE]

```

**Review by KTibow — CHANGES_REQUESTED** [review]

**Pull request closed without merging**